### PR TITLE
Add github action checks

### LINKED ISSUE #44: Replace CI with GitHub Actions
This repository is still using CircleCI. We should plan some time to migrate to GitHub Actions.

### PR DESCRIPTION
**Pull request by carkod**
## Done
Replace Circle CI with Github actions

## QA

Checks need to pass

## Issue
Fixes #44

### CODE REVIEW

File: .dotrun.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"yarn": {"dependencies": {}, "installed_packages": {}, "lockfile_hash": "dd07d22cbf9a738bdb68b591bc1bc742"}, "python": {"requirements": "flask==1.1.2\ntalisker[gunicorn,gevent]==0.18.0\n", "installed_packages": ["Flask-1.1.2.dist-info", "pip-21.0.1.dist-info", "contextvars-2.4.dist-info", "statsd-3.3.0.dist-info", "MarkupSafe-1.1.1.dist-info", "Jinja2-2.11.3.dist-info", "Werkzeug-1.0.1.dist-info", "zope.interface-5.3.0.dist-info", "future-0.16.0.dist-info", "immutables-0.15.dist-info", "requests-2.25.1.dist-info", "chardet-4.0.0.dist-info", "idna-2.10.dist-info", "urllib3-1.26.4.dist-info", "zope.event-4.5.0.dist-info", "wheel-0.36.2.dist-info", "talisker-0.18.0.dist-info", "gevent-21.1.2.dist-info", "humanize-3.2.0.dist-info", "certifi-2020.12.5.dist-info", "setuptools-54.0.0.dist-info", "greenlet-1.0.0.dist-info", "itsdangerous-1.1.0.dist-info", "gunicorn-20.1.0.dist-info"]}}
```
Comment (tbille) [inline .dotrun.json:1]: you might want to add this file in the `.gitignore`

File: .github/workflows/pr.yaml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+jobs:
+  run-image:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build image
+        run: DOCKER_BUILDKIT=1 docker build --tag webteam-releases.canonical.com .
```
Comment (tbille) [inline .github/workflows/pr.yaml:14]:
```suggestion
        run: DOCKER_BUILDKIT=1 docker build --tag webteam.canonical.com .
```

File: .github/workflows/pr.yaml [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+      - name: Run image
+        run: |
+          docker run --detach --env SECRET_KEY=insecure_secret_key --network host webteam-releases.canonical.com
```
Comment (tbille) [inline .github/workflows/pr.yaml:18]:
```suggestion
          docker run --detach --env SECRET_KEY=insecure_secret_key --network host webteam.canonical.com
```

File: Dockerfile [hunk 4]
```diff
@@ -21,7 +20,6 @@ RUN rm -rf package.json yarn.lock .babelrc webpack.config.js requirements.txt
```
Comment (tbille) [inline Dockerfile:20]: Add: `ADD . .` to import the code in the image

File: .github/workflows/pr.yaml (hunk 2 above)
Comment (carkod) [reply .github/workflows/pr.yaml:14]: Yeah, but what happens if we have more projects for `webteam.canonical.com` ? I assume this is not gonna be just for releases

Comment (tbille) [reply .github/workflows/pr.yaml:14]: I think for now it's fine to call it like this. We can iterate through his when we face the problem

File: .dotrun.json (hunk 1 above)
Comment (tbille) [reply .dotrun.json:1]: reopening the conversation since not solved.
This file should be removed and added in the `.gitignore`

Comment (carkod) [reply .dotrun.json:1]: Sorry this should be fixed with a rebase, it's already in a previous PR

**Review by tbille — APPROVED** [review]
:rocket:

**Pull request merged**